guidelines/index.html

```diff
@@ -2540,16 +2540,16 @@ <h1 id="title" class="title">Web Content Accessibility Guidelines (WCAG) 2.2</h1
 </dd>
 
             
-            <dt class="new"><dfn data-lt="cognitive function test|Cognitive function test" id="dfn-cognitive-function-test" tabindex="0" aria-haspopup="dialog" data-dfn-type="dfn">Cognitive function test</dfn></dt>
+            <dt class="new"><dfn data-lt="cognitive function test|Cognitive function test" id="dfn-cognitive-function-test" tabindex="0" aria-haspopup="dialog" data-dfn-type="dfn">認知機能テスト</dfn></dt>
 <dd class="new"><p class="change">[New]</p>
    					
-   <p>A task that requires the user to remember, manipulate, or transcribe information. Examples include, but are not limited to:</p>
+   <p>利用者に情報の記憶、操作又は転記を要求するタスク。例には、次のものが含まれるが、これらに限定されない:</p>
    <ul>
-       <li>memorization, such as remembering a username, password, set of characters, images, or patterns. The common identifiers name, e-mail, and phone number are not considered cognitive function tests as they are personal to the user and consistent across Web sites;</li>
-       <li>transcription, such as typing in characters;</li>
-       <li>use of correct spelling;</li>
-       <li>performance of calculations;</li>
-       <li>solving of puzzles.</li>
+       <li>記憶させること、たとえばユーザー名、パスワード、文字の一式、画像、パターンを覚えさせるなど。一般的な識別子である名前、電子メール及び電話番号は、利用者にとって個人的なものであって、ウェブサイト間で一貫しているため、認知機能テストとはみなされない。</li>
```

Comment:
https://github.com/waic/wcag22/pull/44#discussion_r1475708735
```suggestion
       <li>記憶させること、たとえばユーザー名、パスワード、文字の一式、画像、パターンを覚えさせるなど。一般的な識別子である名前、電子メール及び電話番号は、利用者にとって個人特有のものであって、ウェブサイト間で一貫しているため、認知機能テストとはみなされない。</li>
```